pysys/basetest.py

```diff
@@ -458,9 +458,69 @@ def assertPathExists(self, path, exists=True, abortOnError=False):
 		self.addOutcome(PASSED if pathexists(os.path.join(self.output, path))==exists else FAILED, 
 			'Assertion that path exists=%s for "%s"'%(exists, os.path.normpath(path)), 
 			abortOnError=abortOnError)
+
+	def assertEval(self, evalstring, abortOnError=False, **formatparams):
+		"""Perform a validation based on substituting values into 
+		a .format() string with named {} placeholders and then evaluating it with eval. 
+		
+		Example use::
```

Comment:
Did you really mean to have a double-colon ("::") there?

Reply:
that one was deliberate - it's how you introduce a code block in .rst format